AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -371,30 +371,58 @@ - (void)flushQueue {
 
 - (void)checkPendingLogs {
 
-  // Skip checking pending logs if the channel is paused.
-  if (self.paused) {
-    return;
-  }
-
   // If the interval is default and we reached batchSizeLimit flush logs now.
-  if (self.configuration.flushInterval == kMSFlushIntervalDefault && self.itemsCount >= self.configuration.batchSizeLimit) {
+  if (!self.paused && self.configuration.flushInterval == kMSFlushIntervalDefault && self.itemsCount >= self.configuration.batchSizeLimit) {
     [self flushQueue];
   } else if (self.itemsCount > 0) {
     NSUInteger flushInterval = [self resolveFlushInterval];
-    if (flushInterval == 0) {
 
-      // If the interval is over, send all logs without any additional timers.
+    // Skip sending logs if the channel is paused.
+    if (self.paused) {
+      return;
+    }
+
+    // If the interval is over, send all logs without any additional timers.
+    if (flushInterval == 0) {
       [self flushQueue];
-    } else {
+    }
 
-      // Postpone sending logs.
+    // Postpone sending logs.
```

Comment:
Reformat comment, so that `else` has closing brace of initial `if` block:
``` obj-c
} else {

// comment
....
```

Reply:
https://github.com/Microsoft/AppCenter-SDK-Apple/blob/8c2b612de697a78906f5ceb377c74bbf673fb12d/AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m#L306-L307
https://github.com/Microsoft/AppCenter-SDK-Apple/blob/2c5382129d004854d2ed077b5f2ed3206aecf9cf/AppCenter/AppCenter/Internals/Context/Device/MSDeviceTracker.m#L240
https://github.com/Microsoft/AppCenter-SDK-Apple/blob/a63cfd9474b3fa52b4d2a7643d0424cc8db0eee9/AppCenter/AppCenter/Internals/HttpClient/MSHttpClient.m#L149
https://github.com/Microsoft/AppCenter-SDK-Apple/blob/f85f4042cd0c9096d72647f53d1fee7940b09611/AppCenter/AppCenter/Internals/Ingestion/MSIngestionCall.m#L95